packages/components/Notification/src/useNotification.ts

```diff
@@ -0,0 +1,11 @@
+import { NotificationProps } from './Notification.types';
+
+export const useNotification = (props: NotificationProps): NotificationProps => {
+  const { variant, endText, ...rest } = props;
+
+  return {
```

Comment:
It doesn't seem like this function is needed, since it seems like the object is just being recreated but none of the properties are changing... Is there a plan to add more logic in here in a future change?

Reply:
This was created by the component generator and will most likely eventually contain more logic but it can be removed for this initial pr.

Reply:
I think it's fine to check in as is if there's future plans for using the function.